homeassistant/components/vevor_heater/const.py

```diff
@@ -0,0 +1,8 @@
+"""Constants for the Vevor Heater integration."""
+from datetime import timedelta
+from uuid import UUID
+
+DOMAIN = "vevor_heater"
+
+CHAR_UUID_HEATER_CONTROL = UUID("0000ffe1-0000-1000-8000-00805f9b34fb")
```

Comment:
Please move specific uuids to the pypi code and import instead

Reply:
It's not used in HA Python code (leftover from having everything in the HA repo) so just removed it. I assume it's ok to keep it in the json manifest for discovery purposes.

Reply:
Yes definitely fine in the manifest.  In general, anything that's protocol details / magic numbers for the device should be in the pypi code, but that doesn't apply to the manifest

Reply:
It looks like it didn't actually get removed

Reply:
It should be now, sorry!